packages/plugin-bm-ui/src/components/step/Appearance.tsx

```
@@ -82,10 +81,10 @@ const Appearance = (props: Props) => {
       };
 
       setUiOptions(newUiOptions);
-      onChangeFunction('uiOptions', newUiOptions);
+      onChangeFunction("uiOptions", newUiOptions);
     };
 
-    const defaultValue = (uiOptions['texts'] || {})[key];
+    const defaultValue = (uiOptions["texts"] || {})[key];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)